helpers/movimientosArkaHelper/movimientosArkaHelper.go

```diff
@@ -128,6 +160,29 @@ func PostMovimiento(movimiento *models.Movimiento) (movimientoR *models.Movimien
 	return res, nil
 }
 
+func PostSoporteMovimiento(soporte *models.SoporteMovimiento) (soporteR *models.SoporteMovimiento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "PostSoporteMovimiento hace tal cosa"
```suggestion
// PostSoporteMovimiento ...
func PostSoporteMovimiento(soporte *models.SoporteMovimiento) (soporteR *models.SoporteMovimiento, outputError map[string]interface{}) {
```